lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestSecurityToken.cs

```diff
@@ -54,15 +55,50 @@ public class TestSecurityToken
         private const string Password2 = "none";
 
         /// <summary>
-        /// This is to test pass tokens from client for Yarn application
+        /// This is to test pass multiple tokens from client for Yarn application
         /// </summary>
         [Fact]
         [Trait("Environment", "Yarn")]
         [Trait("Priority", "1")]
         [Trait("Description", "Run CLR Test on Yarn")]
         public void TestSetMultipleSecurityToken()
         {
-            IInjector injector = TangFactory.GetTang().NewInjector(GetRuntimeConfiguration());
+            IInjector injector = TangFactory.GetTang().NewInjector(GetRuntimeConfigurationForMultipleTokens());
+            TestRun(injector);
+        }
+
+        /// <summary>
+        /// This is to test write token in old approach for backward compatibility checking
+        /// </summary>
+        [Fact]
+        [Trait("Environment", "Yarn")]
+        [Trait("Priority", "1")]
+        [Trait("Description", "Run CLR Test on Yarn")]
+        public void TestSecurityTokenBackword()
```

Comment:
Please rename to `TestSecurityTokenBackwardCompatibility()`
Also, add proper `[System.Obsolete(...)]` attribute mentioning JIRA and REEF release in which it will be removed.